### ISSUE TITLE
First translation name is removed when adding question to library from formbuilder

### ISSUE THREAD

**Issue by tinok** [kobotoolbox/kpi#2731]
## Description

When adding a question to the library containing multiple translations (label languages) the POST request does not include the name of the first translation. 

## Steps to Reproduce

1. Upload this form ([d004.xlsx](https://github.com/kobotoolbox/kpi/files/4936839/d004.xlsx))
2. Open it in the formbuilder
3. Click on the 'add to library' button
4. Check the Form Data that was sent in the network POST request

## Expected behavior 

Question is added to the library containing the same translations (label::Français, label::Sango). 

## Actual behavior 

The question stored to the library only includes `label`and `label::Sango`. See form data:

![image](https://user-images.githubusercontent.com/673511/87771280-267b9a80-c7ee-11ea-9c7a-c9ad92def0e6.png)

**Comment by jnm** [comment]
If we drill down into the reproduction steps:

1. <sub>Upload this form ([d004.xlsx](https://github.com/kobotoolbox/kpi/files/4936839/d004.xlsx))</sub>
2. Open it in the formbuilder
    * nullify happens here (`label::Français` becomes `label`, and to the form builder, `label::Sango` is just a meaningless, extra column)
3. Click on the 'add to library' button
    * the form builder copies what it has (i.e. `label` and `label::Sango`) to the library
        * how does this happen? this must be where we can use the unnullify logic on the question that's about to be saved as a new library asset
4. <sub>Check the Form Data that was sent in the network POST request</sub>
5. This is not a part of original steps, but consider that we save the original form now, in the form builder:
    * unnullify happens: `label` is restored to `label::Français`

in short, the form builder only knows about the labels in `label`. for whatever labels should appear in the form builder, _other KPI code_ has to put those labels into `label` (i.e. the null translation, not `label::anything_else`) __before__ passing the JSON to the form builder.

yes, `label::Sango` etc. do appear in the form builder, but only because the form builder passes any columns it doesn't recognize through to the question settings that it displays. a form could have a column `onomatopoeia` and the form builder would dutifully show it.
